### ISSUE TITLE
No notice/error-message appearing

### ISSUE THREAD

**Issue by inovek** [AtomLinter/Linter#478]
Installed 'linter', 'linter-php' and 'linter-phpcs' and it seems to work. But the error/notification message is missing (on the bottom statusbar or tooltip).

![atom_linter_php - no tooltip](https://cloud.githubusercontent.com/assets/1033171/6823124/8308cb00-d2e5-11e4-870d-cc184390dd03.png)

What I've done wrong? I think the mistake is on my side.

**Comment by hd-deman** [comment]
looks like white text on white background ::)

**Comment by inovek** [comment]
Nice idea :)

I switched back to "Atom Dark" theme and syntax but nothing changes. :(

**Comment by inovek** [comment]
Old post deleted, new one created.

I tried another theme but on the tooltip nothing changed. No text appeared! But it took some time to realize that in the statusbar the line-position was (hard to see but) visible. Changed to custom themes and saw the "line x / col y" which directed to the correct position.

But it seems that the notice/error-message itself don't appears.

This issue appears on "OSX Mavericks", some components was installed via sudo-command. (like PHP_CodeSniffer)
I tried the same setup on Windows 8.1 and there this plugin works well.

I think the mistake is anywhere on my side.
But I would be glad if anyone can help to resolve this issue. (showing directions or bug-reasion-ideas is already help full for me)
